### PR TITLE
loadConf() already calls putAll()

### PR DESCRIPTION
**Pull request by SyBen**
At the end of ConfUtils.loadConf(), you either have a (Config + yaml options) Map or a Config Map. Either way, there is no need for a second putAll().

### CODE REVIEW

**Comment by jnioche** [comment]
Not quite. It is possible to have multiple -conf XXX on the command line, for instance to separate a generic configuration file from a more specific one. The change you suggested would only load the last file specified.

What we could do instead would be to change `ConfUtils.loadConf(resource)` into `ConfUtils.loadConf(Config conf, String resource)` so that it adds to an existing Config object. What do you think?

**Comment by SyBen** [comment]
My bad, I missed that use case. Sounds good to me, As it would match the other methods behavior of the same class.

File: core/src/main/java/com/digitalpebble/storm/crawler/protocol/httpclient/HttpProtocol.java [hunk 1]
```diff
@@ -177,10 +177,10 @@ public BaseRobotRules getRobotRules(String url) {
 
     public static void main(String args[]) throws Exception {
         HttpProtocol protocol = new HttpProtocol();
+	Config conf = new Config();
```
Comment (jnioche) [inline core/src/main/java/com/digitalpebble/storm/crawler/protocol/httpclient/HttpProtocol.java]: formatting? you can apply the formatting rules with `mvn java-formatter:format'

**Comment by jnioche** [comment]
Merci Sylvain. Could you please squash your commits after applying the formatting rules? Thanks!

**Comment by jnioche** [comment]
Merged, thanks!

**Pull request merged**